### ISSUE TITLE
Utilize Javascript to add Soundcloud embeds

### ISSUE THREAD

**Issue by PersonMeetup** [PersonMeetup/personmeetup-web#6]
While this idea was something I binned earlier, noticing how Twitter implements this idea caught my attention.
```html
<blockquote class="twitter-tweet">
  <p lang="en" dir="ltr">Sunsets don&#39;t get much better than this one over <a href="https://twitter.com/GrandTetonNPS?ref_src=twsrc%5Etfw">@GrandTetonNPS</a>. <a href="https://twitter.com/hashtag/nature?src=hash&amp;ref_src=twsrc%5Etfw">#nature</a> <a href="https://twitter.com/hashtag/sunset?src=hash&amp;ref_src=twsrc%5Etfw">#sunset</a> <a href="http://t.co/YuKy2rcjyU">pic.twitter.com/YuKy2rcjyU</a></p>&mdash; US Department of the Interior (@Interior) <a href="https://twitter.com/Interior/status/463440424141459456?ref_src=twsrc%5Etfw">May 5, 2014</a>
</blockquote>
<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```
It displays the tweet within HTML blockquotes, turning it into a tweet if Javascript is enabled. Ideally, I should replicate this functionality with my [Soundcloud embeds](https://personmeetup.ca/portfolio/music/).

**Comment by PersonMeetup** [comment]
Some interesting developments happened today! This all stems from Bryce Wray's article ["More tips for using Giscus"](https://www.brycewray.com/posts/2022/07/more-tips-using-giscus/) which covered the use of [`<details>` element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details). Combined with setting the Soundcloud `<iframe>` to `loading="lazy"`, this solves the problem I was addressing in this issue without resorting to Javascript! The only downside is that this [doesn't work on Firefox or Safari](https://www.brycewray.com/posts/2022/05/tips-using-giscus/#get-lazy), but given Chrome's marketshare it shouldn't make much of a difference.

Changes are currently in effect in the portfolio branch, will wait till that's pulled into master before I close.

**Comment by PersonMeetup** [comment]
Issue can finally be closed with the merging of #9!
